SU-HKKU/cardinal_service_profile
# CSD-410 Updates from September QA/Style bugs

# READY FOR REVIEW

# Summary
- Updates from September QA/Style bugs

# Needed By (Date)
- 09/03

# Urgency
- medium

# Steps to Test

1. Pull this branch
2. Clear cache
3. Check for the following changes:
- Individual Opportunity page tags should now appear inline
- Spacing between stories should be increased
- Spacing above Related Opportunities and Resources should be increased
- Spacing below Related Opportunities and Resources cards should be increased
- Opportunity Row taxonomy tags should now have the same styling

# Associated Issues and/or People
- [CSD-275](https://stanfordits.atlassian.net/browse/CSD-410)
- CSD-405
- CSD-403
- CSD-411
- CSD-412
- CSD-409

- CSD-403 — Opportunity row span flex-10-of-12 will be in another Pull Request.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
